src/lib/user/useIsOperationAllowed.ts

```diff
@@ -0,0 +1,77 @@
+import {
```

Comment:
Not sure if this file should be in the `user` folder. It's tied to the user due to the authorities, but it could also be in `auth`- or `access`- folder. We also need a `useCheckAccess`, that would take a model with `access` fields and check for edit-access etc. But since that is on the model itself, it's a bit different. But Im not sure if these access checks should live with authority checks?